File: week-1-and-2-basic-programming/fizzbuzz.rb
input_number = gets.chomp.to_i

if input_number % 3 == 0 && input_number % 5 == 0
   puts "FizzBuzz"
elsif input_number % 3 == 0
   puts "Fizz"
elsif input_number % 5 == 0
   puts "Buzz"
else
   puts input_number
end

File: software-development/week-8-learn-a-new-language/fizzbuzz-app/src/fizzBuzz.js
class FizzBuzz {
  play(number) {
    if (number % 3 === 0 && number % 5 === 0) {
      return 'FizzBuzz';
    } else if (number % 3 === 0) {
      return 'Fizz';
    } else if (number % 5 === 0) {
      return 'Buzz';
    } else {
      return number;
    }
  }
}

var fizzBuzz = new FizzBuzz();

for (var i = 0; i <= 100; i++) {
  console.log(fizzBuzz.play(i));
}